src/publicLinkAccess.js

```diff
@@ -251,5 +251,23 @@ class PublicFiles {
       return Promise.reject(this.helpers.buildHttpErrorFromDavResponse(result.status, result.body))
     })
   }
+
+  /**
+   * Returns the file info for the given public resource
+   * @param   {string} token public share token - may contain the path as well
```

Comment:
Looking at `list` it looks like only the token may be provided, not the path.

Reply:
It needs to contain also path otherwise we wouldn't be able to reach the requested file. Also, in the list, it's called actually `tokenAndPath`.

https://github.com/owncloud/owncloud-sdk/blob/42936db99adeec5dd40a4f79128db4c48283cf45/src/publicLinkAccess.js#L41

I can rename it to the same to be consistent.

Reply:
Ah, now I see where is the problem. `getFileUrl` is not used correctly. I'll fix that and adjust the code accordingly.